src/SFA.DAS.ApprenticeCommitments/Application/Commands/ConfirmEmployerCommand/ConfirmEmployerCommand.cs

```diff
@@ -1,23 +1,35 @@
-﻿using System;
-using SFA.DAS.ApprenticeCommitments.Infrastructure.Mediator;
+﻿using SFA.DAS.ApprenticeCommitments.Infrastructure.Mediator;
+using System;
 
 #nullable enable
 
 namespace SFA.DAS.ApprenticeCommitments.Application.Commands.ConfirmEmployerCommand
 {
+    public class ApprenticeCommitmentStatementId
+    {
+        public ApprenticeCommitmentStatementId((Guid apprenticeId, long apprenticeshipId, long commitmentStatementId) id)
+        {
+            ApprenticeId = id.apprenticeId;
+            ApprenticeshipId = id.apprenticeshipId;
+            CommitmentStatementId = id.commitmentStatementId;
+        }
+
+        public Guid ApprenticeId { get; }
+        public long ApprenticeshipId { get; }
+        public long CommitmentStatementId { get; }
+    }
+
```

Comment:
Not sure adding this class adds any value or makes our use any clearer. And if we do end up with type of pattern should this class be stored outside of the ConfirmEmployerCommand?  

We never use the 3 values together, or do we?